getconfig.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  All Rights Reserved.
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#
+#  The contents of this software are proprietary and confidential to RidgeRun,
+#  LLC.  No part of this program may be photocopied, reproduced or translated
+#  into another programming language without prior written consent of
+#  RidgeRun, LLC.  The user is free to modify the source code after obtaining
+#  a software license from RidgeRun.  All source code changes must be provided
+#  back to RidgeRun without any encumbrance.
+
+import cv2
+import logging
+import sys
+import yaml
+
+from time import time
+from time import sleep
+import numpy as np
+import threading
+import curses
+import cv2
+import signal
+import sys
+import argparse
+import yaml
+import os
+import stat
+
+
+_metrics = {}
+
+
+class GetConfigYaml:
```

Comment:
Add basic documentation to this class. At least a simple statement of what it does and what parameters receives

Reply:
Apply this comment to all the classes in the project